src/player.ts

```diff
@@ -31,19 +34,19 @@ export async function getPlayer(
 ): Promise<PlayerOutput> {
   const hero = await this.createHero();
   try {
-    let origin = 'https://play.esea.net/api';
-    let userUrl = `${origin}/users/${eseaProfileId}`;
-    let profileUrl = `${origin}/users/${eseaProfileId}/profile`;
-    let statsUrl = `${origin}/users/${eseaProfileId}/stats?filters[type_scopes]=pug&filters[period_types]=career`;
-    let lastMatchUrl = `${origin}/users/${eseaProfileId}/matches?page_size=1`;
+    const origin = 'https://play.esea.net/api';
+    const userUrl = `${origin}/users/${eseaProfileId}`;
+    const profileUrl = `${origin}/users/${eseaProfileId}/profile`;
+    const statsUrl = `${origin}/users/${eseaProfileId}/stats?filters[type_scopes]=pug&filters[period_types]=career`;
+    const lastMatchUrl = `${origin}/users/${eseaProfileId}/matches?page_size=1`;
 
     this.debug(`Going to ${origin}`);
     const originResponse = await hero.goto(origin, { timeoutMs: this.timeout });
-    const statusCode = originResponse.response.statusCode;
+    const { statusCode } = originResponse.response;
     if (statusCode !== 200) {
       // Check for cloudflare challenge
-      const title = await hero.document.querySelector("title");
-      if(title != undefined && await title.textContent === "Attention Required! | Cloudflare"){
```

Comment:
This call needs to be awaited due to how the [AwaitedDOM ](https://ulixee.org/docs/hero/basic-client/awaited-dom) works in ulixee hero.